extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -19,8 +19,17 @@
 
 var objects = require(process.cwd() + '/extensions/objects/protractor.js');
 
-var customizeInteraction = function() {
-  // There are no customizations.
+var customizeInteraction = async function(elem, customLetters) {
+  expect(await elem.element(
+    by.css('.protractor-test-custom-letters-div')).isPresent()).toBe(true);
+  for (let letter of customLetters) {
+    await elem.element(by.buttonText('abc')).click();
+    let letterIsPresent = await elem.element(by.buttonText(letter)).isPresent();
```

Comment:
This check also doesn't allow for the element to still be loading when this line runs. You can check this with a `try` like this:

```js
let letterIsPresent = true;
try {
  waitFor.presenceOf(element, "This error should be caught.");
} catch (e) {
  letterIsPresent = false;
}
```

Reply:
Alright, so I tried calling the presenseOf function as you've suggested, but the error doesn't seem to get caught for some reason. I tried explicitly calling the function's statements directly and it works. Is that fine?

Reply:
Could you dig a little into why just calling the function doesn't work? I don't see why it would matter whether we call the function or its lines, and we should avoid copying function code instead of calling the function